cdc/redo/writer/writer.go

```diff
@@ -549,8 +550,8 @@ func (l *LogWriter) setMaxCommitTs(tableID int64, commitTs uint64) uint64 {
 }
 
 // flush flushes all the buffered data to the disk.
-func (l *LogWriter) flush() error {
-	err1 := l.flushLogMeta(0, 0)
+func (l *LogWriter) flush(minResolvedTs uint64) error {
+	err1 := l.flushLogMeta(0, minResolvedTs)
```

Comment:
```suggestion
	err1 := l.ddlWriter.Flush()
	err2 := l.rowWriter.Flush()
	err3 := l.flushLogMeta(0, minResolvedTs)
```
Data should be flushed first.